src/plugins/gs-flatpak.c

```diff
@@ -2332,3 +2332,9 @@ gs_flatpak_refine_metadata_from_installation (GsFlatpak *self,
 	gs_appstream_copy_metadata (app, as_app, TRUE);
 	return TRUE;
 }
+
+FlatpakInstallation *
+gs_flatpak_get_installation (GsFlatpak *self)
```

Comment:
So far none the FlatpakInstallation is never exposed outside of GsFlatpak and I'd prefer to keep using this object to abstract the Flatpak stuff. Also, many of the code of GsFlatpak will be changed/removed so it makes sense to add this temporary code there.

The code will also be much simpler and easier to change when we rebase if we just call something like _gs_flatpak_app_is_default_branch_ or something.